Documentation/Sprint 2/reflection.tex

```diff
@@ -0,0 +1,218 @@
+I want to be able to save the url from a room and come back later.
+&
+\begin{enumerate}
+\item Make the API on the backend for the chats.
+\item Make the API on the backend for the rooms.
+\item Let the frontend do API calls to the backend to receive room and chat data.
+\end{enumerate}
+
+&
+
+\begin{enumerate}
+\item Jaap Heijligers
+\item Tim van der Lippe
+\item Gijs Weterings
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 6 hours
+\item 6 hours
+\item 8 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 3 hours
+\item 6 hours
+\item 9 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item Yes
+\item Yes
+\item Yes
+\end{enumerate}
+&
+\begin{enumerate}
+\item Swapped this issue and the next. Tim made this one. We found the communication between the frontend and backend tasks a bit troublesome, as we did not agree on a specific standard until later in the sprint.
+\item Swapped this issue and the next. Jaap made this one. We found the communication between the frontend and backend tasks a bit troublesome, as we did not agree on a specific standard until later in the sprint.
+\item Jan-Willem took over this issue.
+\item Took slightly longer due to miscommunication with Tim and Jaap. We agreed to improve this in the future by sitting down together and agreeing on the interface first.
+\end{enumerate}
+
+\end{tabular}
+\end{table}
+\begin{table}[h]
+\begin{tabular}{|p{3cm}|p{5.0cm}|p{3.0cm}|p{2.2cm}|p{2.2cm}|p{2.0cm}|p{5cm}}
+
+As a teaching assistant I want to read the documention of the system.
+&
+\begin{enumerate}
+\item Update product vision document with TA and information skills teacher feedback.
+\item Write the architecture design document
+\item Update the requirements document with the system changes of the first 2 sprints.
+\end{enumerate} 
+           
+&
+\begin{enumerate}
+\item Tim van der Lippe
+\item Jan-Willem Gmelig Meyling
+\item Gijs Weterings
+\end{enumerate}
+           
+&
+
+\begin{enumerate}
+\item 2 hours
+\item 12 hours
+\item 8 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item 5 hours
+\item 15 hours
+\item 13 hours
+\end{enumerate}
+
+&
+\begin{enumerate}
+\item Yes
+\item Yes
+\item Yes
+\end{enumerate}
+&
+\begin{enumerate}
+\item Jaap made the preface, we took some extra time to find some literature.
+\item Took a bit longer due to extra feedback rounds.
+\item This has been merged with the product planning document, The actual time is the time we took to create and update this full document, not just the MoSCoW bullets.
+\end{enumerate} 
+\end{tabular}
+\end{table}
+
+Problems we encountered:\\
+Writing a document often did not result in creating the glossaries and references subsections. We need to ensure this will become a standard section of writing a document.\\
+Reviewing Pull-Requests was somewhat vague, and often took too long to complete and give feedback. We agreed on a better workflow to make sure we keep the overview in Pull-Requests. This will also be brought up every sprint meeting.\\
```

Comment:
> It was vague because we did not properly assign storypoints for it and the estimated time for an issue was already exhausted before we went into reviewing and not even processing the review comments.
> We have decided to create separate issues for reviewing the issues.